en/editor/publish/publish-web.md

```diff
@@ -1,64 +1,74 @@
+| Polyfills | Build polyfills that support some new features of scripts. Corresponding processing will be done when scripts are packaged. Developers can choose the polyfills needed according to actual needs. For the time being, only **asynchronous functions** are supported, and more functions will be opened in the future. | `polyfills` |
 
 ### Web Mobile
 
-| Option | Explanation | Field Name |
-| :-- | :-- | :-- |
-| **Resource Server Address** | The server address used to download remote resources, see [General Build Options](build-options.md#resource-server-address) for details. | `remoteServerAddress` |
-| **Orientation** | Device orientation, including `Auto`, `Landscape`, `Portrait`. | `orientation` |
-| **Polyfills** | In Creator 3.0, the Build supports some new feature of `polyfills`, mainly when the script is packaged, and the corresponding processing will be done. This option currently supports `Async Functions` and `coreJs`, the developer can choose according to their needs. | `polyfills` |
-| **VConsole** | Whether to use vConsole. vConsole is similar to the mini version of DevTools and is used to aid debugging. | `embedWebDebugger` |
-| Preview QRCode | For scanning previews, see the description below for details. | - |
-| Preview URL | The link for the preview, see the description below for details. | - |
+| Build options | Description | Field name (for command line publishing) |
+| :--- | :--- | :--- |
+| Resource Server Address | The server address used to download remote resources. For details, please refer to the content of the **Resource Server Address** section below. | `remoteServerAddress` |
+| Device orientation | Optional values ​​include **Auto**, **Landscape**, **Portrait** | `orientation` |
+| Polyfills | Build polyfills that support some new features of scripts. Corresponding processing will be done when scripts are packaged. Currently, there are **async Functions** and **coreJs**. Developers can choose the polyfills needed according to actual needs. | `polyfills` |
+| vConsole | Insert the vConsole debugging plug-in. vConsole is similar to the mini version of DevTools for assisting debugging. | `embedWebDebugger` |
+| Preview the QR code | Used to scan and preview, see the introduction below for details |-|
+| Preview URL | Link for preview, see the introduction below for details |-|
+
+- **Resource server address**
+
+  This item is optional and used to fill in the address where the resource is stored on the server.
+
+    - If **do not fill in** this item, the `remote` folder in the release package directory will be packaged into the built game package.
+    - If **fill in** this item, it will not be packaged into the game package. Developers need to manually upload the `remote` folder in the release package directory to the resource server address filled in after the build. For details, please refer to [Upload resources to remote server](../../asset/cache-manager.md).
```

Comment:
```suggestion
    - If **fill in** this item, it will not be packaged into the game package. Developers need to manually upload the `remote` folder in the release package directory to the resource server address filled in after the build. For details, please refer to [Upload resources to a remote server](../../asset/cache-manager.md).
```